Tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -85,7 +85,7 @@ public function configurations()
                     'de' => array('en', 'fr'),
                     'fr' => array('en', 'de'),
                 ),
-                'locale_fallback' => 'hardcoded',
+                'locale_fallback' => 'merge',
```

Comment:
sorry, i think this one must be "hardcoded". the [single](https://github.com/catharsisjelly/DoctrinePHPCRBundle/blob/issue/255-default-locale-fallback/Tests/Resources/Fixtures/config/single.yml#L48) configuration file tests setting this to a manual value. multiple does not specify anything, so its using the default.